editor/import/3d/post_import_plugin_skeleton_rest_fixer.cpp

```diff
@@ -53,6 +54,15 @@ void PostImportPluginSkeletonRestFixer::get_internal_import_options(InternalImpo
 	}
 }
 
+Variant PostImportPluginSkeletonRestFixer::get_internal_option_visibility(InternalImportCategory p_category, bool p_for_animation, const String &p_option, const HashMap<StringName, Variant> &p_options) const {
+	if (p_category == INTERNAL_IMPORT_CATEGORY_SKELETON_3D_NODE) {
+		if (p_option == "retarget/rest_fixer/keep_global_rest_on_leftovers") {
+			return bool(p_options["retarget/rest_fixer/overwrite_axis"]);
+		}
+	}
+	return true;
+}
+
```

Comment:
It looks a bit inconsistent with other similar parts, but since the result is not different, it should not be a problem.